PSReadLine/Render.cs

```diff
@@ -51,6 +51,7 @@ class RenderData
             public RenderedLineData[] lines;
         }
 
+        private const char ELLIPSIS = '\u2026';
```

Comment:
What about terminals that don't support Unicode?

Reply:
I haven't seen a modern terminal that doesn't support it including Win7.

Reply:
I booted a bare metal Arch iso and it seems that does not support Unicode:

![image](https://user-images.githubusercontent.com/2148248/66879706-2b1bd580-ef74-11e9-9006-a399b6df883d.png)

Reply:
Ok, I'll change it back.